hack/lib/golang.sh

```diff
@@ -334,7 +334,7 @@ EOF
   local go_version
   go_version=($(go version))
   local minimum_go_version
-  minimum_go_version=go1.8.3
+  minimum_go_version=go1.9
```

Comment:
If there are no compatibility issues (e.g., K8S import some packages only in go1.9), maybe we'd better keep `minimum_go_version` to be the last version we used. @cblecker WDYT?

Reply:
Yeah, you're right, we don't need to bump the minimum just yet.